fee-monitor/src/index.ts

```diff
@@ -172,7 +171,7 @@ async function main() {
             reports.push(`<h3>Stakeholders</h3>`);
             reports.push(`<ul>${stakeholders.join("<br />\r\n")}</ul>`);
 
-            email.sendInfo("is working.", `${reports.join("\r\n")}`);
```

Comment:
The message's format is HTML. Is it okay?

Reply:
I removed HTML tags